### PR TITLE
Expose WebView background color

### PR DESCRIPTION
**Pull request by cangosta**

### CODE REVIEW

File: WebViewControl/GlobalSettings.cs [hunk 1]
```diff
@@ -11,7 +12,16 @@ public class GlobalSettings {
...
         private string userAgent;
         private string logFile;
-        private string cachePath = Path.Combine(Path.GetTempPath(), "WebView" + Guid.NewGuid().ToString().Replace("-", null) + DateTime.UtcNow.Ticks);
+        private string cachePath = Path.Combine(Path.GetTempPath(), "WebView" + Guid.NewGuid().ToString().Replace("-", null) + DateTime.UtcNow.Ticks);
+        private Color? backgroundColor;
+
+        public Color? BackgroundColor {
+            get => backgroundColor;
+            set {
+                EnsureNotLoaded(nameof(BackgroundColor));
```
Comment (cangosta) [inline WebViewControl/GlobalSettings.cs:21]: @joaompneves in the case of the background color, this check is not needed, right?

**Review by joaompneves — COMMENTED** [review]
isnt the default color transparent? if it was transparent we could leave this feature out of the webview

**Comment by marianacapelo** [comment]
> isnt the default color transparent? if it was transparent we could leave this feature out of the webview

It is. But the default of BackgroundColor property on CefSettings is white. I just committed the change to use the Settings there, can you please review @joaompneves ?

**Comment by joaompneves** [comment]
Lets simplify and make it transparent by default a and leave the background color setting out of the webview

**Comment by marianacapelo** [comment]
> Lets simplify and make it transparent by default a and leave the background color setting out of the webview

Unfortunately it does not work. BackgroundColor property cannot be transparent, see https://xilium.bitbucket.io/cefglue/doc/html/1694AD49.htm

File: WebViewControl/GlobalSettings.cs [hunk 2]
```diff
@@ -1,4 +1,5 @@
-﻿using System;
+﻿using System;
+using System.Drawing;
```
Comment (joaompneves) [inline WebViewControl/GlobalSettings.cs:2]: lets not depend on system.drawing .... consider creating a Color structure

**Comment by joaompneves** [comment]
> > Lets simplify and make it transparent by default a and leave the background color setting out of the webview
> 
> @joaompneves Unfortunately it does not work. BackgroundColor property cannot be transparent, see https://xilium.bitbucket.io/cefglue/doc/html/1694AD49.htm

Thats a very old docs... although looking at more recent ones it appears thats still true.

**Pull request closed without merging**